ios/emerald_mobile_wallet/main.m

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
BSD

Reply:
All iOS apps use this... Not sure we can get away with not having it

Reply:
This is generated by the React-Native platform

Reply:
React-Native is not under MIT license, though they just put React under MIT